packages/shared/lib/utils/encryption.manager.ts

```diff
@@ -19,40 +18,15 @@ interface DataRecordJson {
 
 export const pbkdf2 = utils.promisify(crypto.pbkdf2);
 export const ENCRYPTION_KEY = process.env['NANGO_ENCRYPTION_KEY'];
+if (!ENCRYPTION_KEY) {
+    throw new Error('NANGO_ENCRYPTION_KEY environment variable is required.');
+}
 
-class EncryptionManager {
-    private key: string | undefined;
-    private algo: CipherGCMTypes = 'aes-256-gcm';
-    private encoding: BufferEncoding = 'base64';
-    private encryptionKeyByteLength = 32;
+class EncryptionManager extends Encryption {
```

Comment:
ideally all functions in this file should be moved in the model they refer to. I kept it for simplicity sake, I didn't want to go down the rabbit hole of refactoring a huge chunk of the codebase